docs-kits/kits/Manufacturing as a Service Kit/Software Development View/page_software_architecture.md

```diff
@@ -0,0 +1,858 @@
+| Keycloak                                            | Keycloak is an open source software which handels the identity and access management (IAM) for the Manufacturing Network Registry.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| API Gateway                                         | A security gateway protecting the exposed interfaces from AMS and Search Engine by proxying them and controlling access to them.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+
+#### 5.2.2 Marketplace / MaaS Buyer Application - MaaS Portal
+
+The Siemens MaaS Portal is the prototypical implementation of a federated marketplace in CDME. It is a cloud-based Mendix application and connects offer and demand within a manufacturing network of networks. The prototypical environment also acts as the MaaS Buyer Application. A user account management organizes the access and menu workflow. The MaaS Portal further offers a request configuration, including a user account-specific request history. After searching for potential suppliers, the graphical user interface plots a list of suppliers or shows them in a map view, including all filter criteria information for supplier selection and quotation request. In addition, users can obtain information about the network partners and the onboarding process. The idea is to also offer a marketplace for digital services, supporting the order management and execution process.
```

Comment:
@HFocken I don't understand why a reference implementation shouldn't be company owned. Our architecture was implemented using the low-code platform Mendix from Siemens (which is the only reference to a commercialized product). The MaaS portal itself is not a commercial product but just a demonstrator.

Reply:
A reference implementation should never be company owned and this would be the only case where this would be done that way (according to our knowledge) - we established the category "success stories" for that reason. Reference implementations should not be branded but as open and accessible as possible.

You can highlight that you used Mendix to establish the reference implementation, but otherwise:

1) Either remove the brand name from the Reference implementation title here
2) Move the whole section to "success stories"

Thank you!

Reply:
Content has been adapted in new request: https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/898/commits/614e0eae41793274448840e07d529996e415e57c

/cc @nbarysch @maximilianong @danielmiehle 

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/898

Reply:
Thank you!